doc/config.toml

```diff
@@ -257,7 +257,7 @@ url = "http://www.apache.org/security/"
   desc = "Chat with other project developers at #avro channel"
 [[params.links.developer]]
   name = "Developer mailing list"
-  url = "mailto:dev@avro.apache.org"
+  url = "https://lists.apache.org/list.html?dev@avro.apache.org"
```

Comment:
If we do this then we have to do it for `user@` too (https://github.com/apache/avro/pull/2555/files#diff-8cb6d82f590d5eac7989519b7da2ebbe792c228aa3a402be1bc2fa1aa0ba80ccR229).

Reply:
Great suggestion!